public/js/safetymaps/modules/vrh/vrh-waterwinning.js

```diff
@@ -70,12 +71,22 @@ dbkjs.modules.vrh_waterwinning = {
                         me.newIncident(incident);
                     });
                     $(dbkjs.modules.incidents.controller).on("end_incident", function () {
-                        me.resetTab();
+                        me.afterRequestIsDone(me.resetTab.bind(me));
                     });
                 }
             });
         }
     },
+    afterRequestIsDone: function (callback) {
+        var me = this;
+        if (me.requestIsBuzy) {
+            window.setTimeout(me.afterRequestIsDone, 100);
```

Comment:
This loses the callback parameter and busy is spelled wrong. Also it's very ugly, ideally the waterwinning info request should be canceled.

Reply:
@matthijsln see next commit.

Reply:
Looks better!